### DISCUSSION TITLE
External metadata support

### DISCUSSION THREAD

**Discussion by yatounoneko** [gotson/komga#1734] [Ideas]
I hope to load external metadata files (e.g., .opt[#1575 Mentioned in the discussion](https://github.com/gotson/komga/discussions/1575#discussioncomment-9661279)) with the same name in the same folder as the comic files (e.g., .pdf, .cbr/.cbz, .zip, .rar). 

I have looked at #277 & #1240 and noticed that most PDFs contain really crappy metadata. Some PDF file from TW online comics providers also lack accurate metadata. 

However, I can import these files into Calibre, edit the metadata, and then output metadata files with the same name (.opt).

So is this a feasible method? Similar to how subtitles are used as external files for videos. ( Similar to request #1713 )

**Comment by gotson** [comment]
Duplicate of #1713
